onnxruntime/core/optimizer/attention_fusion.cc

```diff
@@ -280,8 +281,7 @@ Status AttentionFusion::ApplyImpl(Graph& graph, bool& modified, int graph_level,
           reshape_count++;
         }
       }
-
-      if (add_count == 1 && matmul_count == 3) { // BERT
+      if (add_count == 1 && matmul_count == 3) { // BERT or DistilBert
```

Comment:
It is better to count Shape, and add condition that shape_count == 0 or shape_count == 2.